components/json_path.rst

```diff
@@ -0,0 +1,251 @@
+The JsonPath Component
+======================
+
+.. versionadded:: 7.3
+
+    The JsonPath component was introduced in Symfony 7.3 as an
+    :doc:`experimental feature </contributing/code/experimental>`.
+
+The JsonPath component provides a powerful way to query and extract data from
+JSON structures. It implements the `RFC 9535 (JSONPath) <https://datatracker.ietf.org/doc/html/rfc9535>`_
+standard, allowing you to navigate complex JSON data with ease.
+
+Just as the :doc:`DomCrawler component </components/dom_crawler>` allows you to navigate and query HTML/XML documents
+using CSS selectors, the JsonPath component provides a similar fluent interface to traverse and search JSON structures
```

Comment:
JsonPath doesn't provide fluent interfaces, so I think this should reworded as well